packages/react-core/src/components/Wizard/WizardFooter.tsx

```diff
@@ -33,6 +35,12 @@ export interface WizardFooterProps {
   isBackHidden?: boolean;
   /** Flag to hide the cancel button */
   isCancelHidden?: boolean;
+  /** Additional props for the Next button. */
+  nextButtonProps?: Omit<FooterButtonProps, 'isDisabled' | 'type'>;
+  /** Additional props for the Back button. */
+  backButtonProps?: Omit<FooterButtonProps, 'isDisabled'>;
```

Comment:
Is there a reason for including `type` for the nextButtonProps but not backButtonProps?

Reply:
I am omitting that `type` prop, because it is being set to `type="submit"` on the nextButton, so I thought it may not be a good idea to have a possibility to overwrite it.